src/libcharon/plugins/kernel_super/oper_db_client/liboper_db_client/liboper_db_client.h

```diff
@@ -45,7 +45,7 @@ bool liboper_db_client_set(liboper_db_obj_t* objs[], int n_obj, int flags);
 liboper_db_obj_t* liboper_db_client_db_obj_new(const enum LIBOPER_DB_CLIENT_CLASS_IDS class_id, const char* new_key[], uint32_t n_keys);
 void liboper_db_client_db_obj_destroy(liboper_db_obj_t* obj);
 
-bool liboper_db_client_db_obj_enum_set(liboper_db_obj_t* obj, const enum LIBOPER_DB_CLIENT_FIELD_IDS field_id, uint32_t enum_value);
+bool liboper_db_client_db_obj_enum_set(liboper_db_obj_t* obj, const enum LIBOPER_DB_CLIENT_FIELD_IDS field_id, enum LIBOPER_DB_CLIENT_FIELD_ENUM_VALUES enum_value);
```

Comment:
Make sure to also update `oper_db_client_db_obj_t::enum_set()` in `oper_db_client.h` and `sl_oper_db_client.c`